src/components/Footer/index.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+import './Footer.css';
+import LOGO from '../../assets/img/craneLogo.png';
+
+const Footer = () => (
+  <React.Fragment>
```

Comment:
you could use the shorthand for `React.Fragment` - `<></>`

Reply:
I tried to make the code as easy as possible for people who are not very familiar with React syntax, I did this with Henry in mind, so he can easily understand the code.